### DISCUSSION TITLE
Extend SMILES format contains both chirality and pseudoatoms,not working

### DISCUSSION THREAD

**Discussion by niuzhenjiang** [epam/ketcher#1269] [Q&A] [answered]
Extend SMILES format contains both chirality and pseudoatoms,not working and
I can't directly paste special for smiles with extend SMILES to chemdraw software. Is there any way to preserve chirality and pseudo-atom information while still using chemdraw
![image](https://user-images.githubusercontent.com/42111602/154393602-c52aa91b-1501-438c-aced-c105f9b08436.png)
![image](https://user-images.githubusercontent.com/42111602/154393649-6787097e-e0ff-45b8-a3fd-e168610c6ffa.png)

**Comment by ElenaOdnoshivkina** [comment, marked as answer]
Hi @niuzhenjiang 

Extended SMILES format allows you to save information about chirality and pseudo atoms. The function of saving the file in extended SMILES format in Ketcher works correctly.  It's possible that ChemDraw software doesn't support Extended SMILES format, so could you clarify what exactly is your problem?

**Comment by niuzhenjiang** [comment]
Sorry for my unclear expression, I just want to confirm how the format of extend smiles can be used in chemdraw software, your answer already has the answer I want, it seems that it cannot be used directly in chemdraw software, thank you very much
